app/src/ui/history/commit-summary.tsx

```diff
@@ -374,24 +375,23 @@ export class CommitSummary extends React.Component<
             )}
 
             {enableSideBySideDiffs() && (
-              <>
-                <li
-                  className="commit-summary-meta-item without-truncation"
-                  title="Split View"
```

Comment:
This title doesn't make sense, it controls both Split View and Whitespace

<img width="1321" alt="Screen Shot 2021-03-25 at 12 31 11 AM" src="https://user-images.githubusercontent.com/7467062/112419372-b2d94800-8d01-11eb-9bb4-1937b85cc6cb.png">

Reply:
Nice catch!